examples/awesome-vault-service/operator/main.go

```diff
@@ -113,7 +60,7 @@ func main() {
 	// 6. Build the operator, providing operator config, the response calculator and a task response hashing
 	// function, and then start it.
 	// We leave this last parameter as nil because we are using the default hashing function provided by the SDK
-	operator, err := operator.NewOperatorFromConfig(operatorConfig, possibleFailureCalculator, nil)
+	operator, err := operator.NewOperatorFromConfig(operatorConfig, possibleFailureCalculator, nil, logger, taskManagerAbi)
```

Comment:
Let's reorder these parameters so all v2 modules take logger first, then the config, then the ABI, and last any other parameters